src/app/date-widget/page.jsx

```diff
@@ -43,10 +43,10 @@ function DateWidgetPage() {
           <DateTimeWidget config={dateWidgetConfig}/>
         </div>
       </div>
-      {!hideNav && <div>
+      {!hideNav && <div className="h-24 w-full">
         <p>URL to copy</p>
         <Suspense>
-        <p className=' sm:text-xl text-sm'>{copyUrl}</p>
+        <p className='sm:text-xl text-sm h-full'>{copyUrl}</p>
```

Comment:
doing this will cause the url to be hidden like so
http://localhost:3000/date-widget?hidenav=1&tz=Asia/Calcutta&locale=en-US&time=0&date=0&day=0&quarter=1...

notice the "..." ? we don't want that